Dockerfile

```diff
@@ -1,24 +1,39 @@
-FROM debian:jessie
+FROM debian:stretch-slim
 
-ARG DOWNLOAD_URL="https://s3-us-west-2.amazonaws.com/grafana-releases/master/grafana_latest_amd64.deb"
+ARG GRAFANA_VERSION="latest"
+ARG GRAFANA_URL="https://s3-us-west-2.amazonaws.com/grafana-releases/release/grafana-$GRAFANA_VERSION.linux-x64.tar.gz"
+ARG GF_UID="472"
+ARG GF_GID="472"
 
```

Comment:
IMHO you can add build argument for architecture - see https://github.com/monitoringartist/grafana-xxl/pull/8/files. Then you can build also arm images easily.

Reply:
@jangaraj It's a bit out of scope for this PR but that's definitely something to look into when we have inhouse ARM builds of Grafana.